test/surface/component_test.exs

```diff
@@ -105,18 +105,14 @@ defmodule Surface.ComponentTest do
     prop id, :string
     data id_copy, :string
 
-    @impl true
-    def update(assigns, socket) do
-      socket =
-        socket
-        |> assign(assigns)
-        |> assign(:id_copy, assigns.id)
-
-      {:ok, socket}
+    def update(assigns) do
```

Comment:
```suggestion
    defp update(assigns) do
```
I suggest to make this function private in order to avoid that users believe this is part of the `Surface` api neither `PhoenixLiveView` api when looking at the tests.